t/app/controller/report_inspect.t

```diff
@@ -94,11 +96,6 @@ FixMyStreet::override_config {
     };
 
     subtest "test duplicate reports are shown" => sub {
-        my ($report2) = $mech->create_problems_for_body(1, $oxon->id, 'The other report is a duplicate of this one', {
-            category => $report->category, cobrand => 'fixmystreet', areas => ',2237,2420',
-            whensent => \'current_timestamp',
-            latitude => 51.847694, longitude => -1.355909,
-        });
         my $old_state = $report->state;
```

Comment:
This test isn't really testing anything any more, because now both reports have the same title :)